src/commands/ReminderCommand.ts

```diff
@@ -18,49 +18,36 @@ class ReminderCommand extends Command {
 	}
 
 	startScheduler(client: Client) {
-		schedulerTick(client);
-		cron.schedule("*/30 * * * *", schedulerTick.bind(this, client));
+		reminderSchedulerTick(client);
+		// every 30 minutes
+		cron.schedule("*/30 * * * *", reminderSchedulerTick.bind(this, client));
 	}
 
 	async execute(interaction: ChatInputCommandInteraction): Promise<void> {
 		const subcommand = interaction.options.getSubcommand();
 
-		if (!interaction.guild) {
-			await interaction.reply("This command can only be used inside a guild.");
-			return;
-		}
+		const commands: { [key: string]: (interaction: ChatInputCommandInteraction) => Promise<void> } = {
+			set: reminderSet,
+			delete: reminderDelete,
+			list: reminderList,
+		};
 
-		switch (subcommand) {
-			case "set": {
-				reminderSet(interaction);
-				break;
-			}
-			case "delete": {
-				reminderDelete(interaction);
-				break;
-			}
-			case "list": {
-				reminderList(interaction);
-				break;
-			}
-		}
+		await commands[subcommand](interaction);
```

Comment:
You doing a check if a command exists [last comment](https://github.com/Cowoding-Jams/Jambo/pull/53#pullrequestreview-1248033944) but here not

Reply:
Yeah removed it for the proposal command